tests/integration-tests/conftest.py

```diff
@@ -837,7 +842,7 @@ def _create_vpc_stack(request, template, region, cfn_stacks_factory):
     return stack
 
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope="class")
```

Comment:
To make sure the teardown order is correct. For example, the code here should ensure s3_bucket_factory is teared down after cluster_factory. This requires the scope of s3_bucket_factory to be greater or equal to the cluster_factory. If the two scopes are equal, s3_bucket_factory should appear in front of cluster_factory.
This is to avoid failures during the final update to delete the log